django_cloudflare_push/middleware.py

```diff
@@ -50,6 +62,21 @@ def sort_urls(urls):
     return urls
 
 
+def create_header_content(urls):
+    """
+    Creates the content for the Link header.
+    """
+    links = []
+    for url in urls[:10]:
+        link = "<%s>; rel=preload" % (url, )
```

Comment:
Would this be a bit more readable perhaps?:

```python
ext = url.rsplit(".")[-1].lower()
if ext in EXTENSION_AS:
    link = "<%s>; rel=preload; as=%s" % (url, EXTENSION_AS[ext])
else:
    link = "<%s>; rel=preload" % (url,)
```

Reply:
That's a lot more readable, yes. (Been busy.)